CopyKeys.ps1

```diff
@@ -910,7 +912,7 @@ function Start-CopyKeys
     $TargetBekVault = $UserInputs["TargetBekVault"]
     $TargetKekVault = $UserInputs["TargetKekVault"]
 
-    $Context = Get-AzureRmContext
+    $Context = Get-AzContext

```

Comment:
Again if the Login changes are made, you could add another check here to prevent a call.

    if($Context -eq $null)
    {
        $Context = Get-AzContext
    }